docs/documentation/mission_module/mission_file/segments.rst

```diff
@@ -389,12 +408,141 @@ Typically, it will be used as last segment to compute a reserve based on the Zer
+    wheels_friction: 0.03
+    target:
+      equivalent_airspeed:
+        value: data:mission:operational:takeoff:Vr
+
+.. seealso::
+
+    Python documentation: :class:`~fastoad.models.performances.mission.segments.registered.ground_speed_change.GroundSpeedChangeSegment`
+
+.. _segment-rotation:
+
+:code:`rotation` segment
+========================
+
+.. versionadded:: 1.5.0
+
+This segment is used to represent a rotation while still on the ground. This segment is specifically used for takeoff.
+The specific keys are (in addition to wheel friction coefficient):
+
+:code:`rotation_rate` in (rad/s) is the rotation speed used to realise the manoeuvre (by default 3deg/s, compliant with CS-25 )
+
+:code:`alpha_limit` (in rad) is the maximum angle of attack possible before tail strike (by default 13.5deg).
+
+The segment ends when lift equals weight. Therefore, no target needs to be set.
+
+**Example:**
+
+.. code-block:: yaml
+
+    segment: rotation
+    wheels_friction: 0.03
+    rotation_rate:
+      value: 0.0523
+    alpha_limit:
+      value: 0.3489
+
+.. seealso::
+
+    Python documentation: :class:`~fastoad.models.performances.mission.segments.registered.takeoff.rotation.RotationSegment`
+
+.. _segment-end_of_takeoff:
+
+:code:`end_of_takeoff` segment
+==============================
+
+.. versionadded:: 1.5.0
+
+This segment is used at the end of the takeoff phase, between lift off and before reaching the safety altitude. The target sets the safety altitude.
+Because this phase is quite dynamic, it is a good practice to lower the time step at least to 0.05s for a
+good accuracy on takeoff distance.
+
+**Example:**
+
+.. code-block:: yaml
+
+    segment: end_of_takeoff
+    time_step: 0.05
+    target:
+      delta_altitude:
+        value: 35
+        unit: ft
+
+.. seealso::
+
+    Python documentation: :class:`~fastoad.models.performances.mission.segments.registered.takeoff.end_of_takeoff.EndOfTakeoffSegment`
+
+.. _segment-takeoff:
+
+:code:`takeoff` segment
+=======================
+
+.. versionadded:: 1.5.0
+
+This segment is the sequence of `ground_speed_change segment`_, `rotation segment`_ and `end_of_takeoff segment`_.
+
+The parameters for this segment are the same as for its 3 components, except that:
+
+  - :code:`time_step` is used only for `ground_speed_change segment`_ and `rotation segment`_.
+  - time step for `end_of_takeoff segment`_ is driven by the additional parameter :code:`end_time_step`
+  - target speed at end of `ground_speed_change segment`_ is driven by the additional parameter :code:`rotation_equivalent_airspeed`
+  - the target of :code:`takeoff` segment is the target of `end_of_takeoff segment`_, meaning it sets the safety altitude.
+
+**Example:**
+
+.. code-block:: yaml
+
+    segment: takeoff
+    wheels_friction: 0.03
+    rotation_equivalent_airspeed:
+        value: data:mission:operational:takeoff:Vr
+    rotation_rate:
+      value: 0.0523
+      units: rad
+    rotation_alpha_limit_truc:
```

Comment:
The name of this parameter sounds wrong, or still in a 'temporary' state.

Reply:
Oops. Fixed in 0939b08c667df62a2f01f51e563f86cab95f131c with some other minor problems.